modules/oe_content_extra_project/config/install/field.field.oe_organisation.oe_cx_project_stakeholder.oe_contact_url.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.oe_organisation.oe_contact_url
+    - oe_content_entity_organisation.oe_organisation_type.oe_cx_project_stakeholder
+  module:
+    - link
+id: oe_organisation.oe_cx_project_stakeholder.oe_contact_url
+field_name: oe_contact_url
+entity_type: oe_organisation
+bundle: oe_cx_project_stakeholder
+label: 'Contact page URL'
+description: ''
```

Comment:
I compared with the original oe_stakeholder bundle, and there the field has a description:
```yml
description: 'URL of the stakeholder contact page.'
```